enhancements/update/synchronized-upgrades.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Synchronized Upgrades Between Clusters
+authors:
+  - "@danwinship"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - TBD
+creation-date: 2021-01-11
+last-updated: 2021-01-11
+tracking-link:
+  - https://issues.redhat.com/browse/SDN-2603
+see-also:
+  - "/enhancements/network/dpu/overview.md"
+---
+
+# Synchronized Upgrades Between Clusters
+
+## Summary
+
+In a [cluster with DPUs](../network/dpu/overview.md) (eg, BlueField-2
+NICs), the x86 hosts form one OCP cluster, and the DPU ARM systems
+form a second OCP cluster. This makes upgrades to new OCP releases
+complicated, because there is currently no way to synchronize upgrades
+between the two clusters, but rebooting the BF-2 systems as part of
```

Comment:
nit: `overview.md` link was helpful, but can we stick to terms from the glossary (like "DPU NIC") instead of using "BF-2" (a specific subset of possible BPU NICs)?